bears/vcs/git/GitRevertInspectBear.py

```diff
@@ -0,0 +1,148 @@
+import re
+
+from difflib import SequenceMatcher
+
+from bears.vcs.git.GitCommitMetadataBear import GitCommitMetadataBear
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.misc.Shell import run_shell_command
+from coalib.results.Result import Result
+
+
+class GitRevertInspectBear(GlobalBear):
+    LANGUAGES = {'Git'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+
+    BEAR_DEPS = {GitCommitMetadataBear}
+
+    GIT_REVERT_COMMIT_RE = re.compile(
+        r'Revert\s\".+\"\n\nThis\sreverts\scommit\s([0-9a-f]{40})\.')
+
+    def clean_inspect_revert_branch(self, branchname):
+        """
+        Delete git branch.
+
+        branchname:     Name of the branch to be deleted.
+        """
+        switch_branch_command = 'git checkout ' + branchname
+        run_shell_command(switch_branch_command)
+        run_shell_command('git branch -D inspectrevertbranch')
+
+    def _check_modified_file_similarity(self, file_path,
+                                        reverted_commit_sha,
+                                        minimum_similarity_ratio):
+        """
+        Compare the changes in file modified by the
+        revert commit with the changes actually
+        expected in the revert commit.
+
+        :param file_path:                   Relative path to the modified
+                                            file.
+        :param reverted_commit_sha:         Commit hash of reverted commit.
+        :param minimum_similarity_ratio:    Minimum similarity ratio
+                                            required by files in revert
+                                            commit.
+        """
+        with open(file_path, 'r') as f:
+            revert_file_content = f.read()
+
+        create_new_branch_command = (
+            'git checkout -b inspectrevertbranch HEAD^')
+        run_shell_command(create_new_branch_command)
+
+        create_expected_revert_commit = ('git revert %s --no-'
+                                         'edit' %
+                                         reverted_commit_sha)
+        _, err = run_shell_command(create_expected_revert_commit)
+        if err:
```

Comment:
add `  # pragma: no cover` after this, so that your build goes green and we can review it